modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin_{context}"]
+= Running your dynamic plug-in
+
+You can run the plugin using a local development environment. The OpenShift console will run in a container connected to the cluster you have logged into.
+
+.Prerequisites
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have an OpenShift cluster running.
+* You must have link:https://www.docker.com/[Docker] or link:https://podman.io/[Podman] installed.
+
+.Procedure
+
+In one terminal window:
```

Comment:
Shall we say 
```
In the local cloned directory,in one terminal window
```

Reply:
What do you think of this:

1. Open two terminal windows in the local directory of your cloned repository.
2. Run the following commands in the first terminal:
3. Run the following commands in the second terminal:

.Verification
- Visit localhost...

Reply:
I like that